### PR TITLE
Release 0.2.7.

### PR DESCRIPTION
**Pull request by bdice**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by bdice** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -18,20 +18,20 @@ build:
 
 requirements:
   host:
-    - python >=3.4
+    - python
```
Comment (isuruf) [inline recipe/meta.yaml]: Why did you remove this?

Comment (bdice) [reply recipe/meta.yaml]: I wasn't thinking straight, I caught the error when I re-read it...

**Comment by bdice** [comment]
@conda-forge-admin, please rerender.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -18,20 +18,20 @@ build:
...
   run:
-    - flask >=0.12
-    - flask-assets
+    - flask >=1.0
+    - flask-assets >=2.0
     - flask-turbolinks
     - jsmin
     - libsass
     - natsort
-    - python >=3.4
```
Comment (isuruf) [inline recipe/meta.yaml:31]: Why remove this?

Comment (bdice) [reply recipe/meta.yaml:31]: @isuruf I was trying to follow the instructions here: https://conda-forge.org/docs/maintainer/adding_pkgs.html#requirements

I don't see `python` listed under the `run` category. It lists `python` under `host` and it lists "most python libraries" under `run` but not `python` which confused me. I will add it back.

**Comment by bdice** [comment]
@conda-forge-admin, please rerender.

**Comment by bdice** [comment]
@conda-forge-admin, please rerender.
